docs/source/configuring.rst

```diff
@@ -4,16 +4,24 @@ Configuring
 Namespace
 ---------
 
-The first step is to create a Serverless instance, which maps to a function namespace.
+The first step is to create a Serverless instance, which maps to a function namespace. All your functions will be deployed to this namespace.
 When deploying, the `scw-serverless` CLI will look for a Serverless instance in the global scope.
 
 .. code-block:: python
 
    import os
    from scw_serverless import Serverless
 
-   app = Serverless("my-namespace", secret={
-      "SCW_SECRET_KEY": os.environ["SCW_SECRET_KEY"]
+   app = Serverless(
+      "my-namespace",
+      env={
+         "NON_CONFIDENTIAL_PARAMETERS": "My Param"
+      },
+      secret={
+         "SCW_SECRET_KEY": os.environ["SCW_SECRET_KEY"],
+         "MY_SECRET_KEY": os.environ["MY_SECRET_KEY"]
+       },
+      gateway_domains=["app.example.com"]
    })
 
 .. autoclass:: scw_serverless.app.Serverless
```

Comment:
- I don't know how we could Highlight that the func() method creates the endpoints
- get() along with the other method should be described in a full section with details on the API gateway config

Reply:
That's a really brilliant idea, it makes sense to group everything API-gateway related into a single big config chapter. Thanks a lot for the suggestion.